blocks/test/fixtures/core-embed.serialized.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:core/embed url="https://example.com/" -->
+<figure>https://example.com/
```

Comment:
On second glance, in our case it could probably even be just part of the `{ url }` bit, i.e. `{ '\n' + url }`.

Not too worried about it though.